packages/libs/src/services/schemaValidator/schemas/playlistSchema.json

```diff
@@ -64,13 +64,7 @@
           "default": null
         }
       },
-      "required": [
-        "playlist_name",
-        "playlist_id",
-        "description",
-        "is_album",
-        "is_private"
```

Comment:
hmm I think this is functionally fine? We probably set default values on discovery if these aren't provided.

However, I do see an argument for leaving them as most clients really ought to know their desired values.

Reply:
do they need to be required though? fwiw sdk onl puts "playlist_name" as required

Reply:
yeah probably not. Depends if we're trying to save devs from themselves or not 🤷